### DISCUSSION TITLE
Title text not aligned to the traffic lights

### DISCUSSION THREAD

**Discussion by Aaron-212** [ghostty-org/ghostty#3868] [Feature Requests, Ideas]
`macos-titlebar-style = tabs`

<img width="812" alt="Frame 1" src="https://github.com/user-attachments/assets/ccc9000b-ba88-417e-bc57-51d4ea25c926" />

imo it looks better when they are aligned

**Comment by JakeWharton** [comment]
~Dupe of #3174~ Actually sorry I didn't see this was for the title, not the output.
